### ISSUE TITLE
Link drop-down not available in metadata text fields

### ISSUE THREAD

**Issue by FAtherden-eLife** [substance/texture#1180]
I am unable to add a link in text fields in the 'Metadata' view. I think this should be possible for text fields. 

If I add a link in the source data then the link text simply displays with underline formatting, with no option to follow the link:
![screen shot 2019-02-18 at 14 13 43](https://user-images.githubusercontent.com/43879983/52956572-69caa300-3387-11e9-98c4-46660d937267.png)

XML:
```<permissions>
        <ali:free_to_read/>
        <license xlink:href="http://creativecommons.org/publicdomain/zero/1.0/">
          <ali:license_ref>http://creativecommons.org/publicdomain/zero/1.0/</ali:license_ref>
          <license-p>This is an open-access article, free of all copyright, and may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose. The work is made available under the <ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/publicdomain/zero/1.0/">Creative Commons CC0 public domain dedication</ext-link>.</license-p>
        </license>
      </permissions>

**Comment by oliver----** [comment]
Yep. Thanks.

**Comment by oliver----** [comment]
The command for editing links seems not be enabled in Metadata view.

**Comment by oliver----** [comment]
Fixed via #1246
